packages/emotion/README.md

```diff
@@ -47,65 +33,324 @@ const myStyle = css`
 app.classList.add(myStyle)
 ```
 
-Look here 👉 _[emotion babel plugin feature table and documentation](https://github.com/emotion-js/emotion/tree/master/packages/babel-plugin-emotion)_
+## API
+
+### css
+
+The `css` function accepts styles as a template literal, object, or array of objects and returns a class name. It is the foundation of emotion.
+
+#### String Styles
+
+```jsx
+// @live
+import { css } from 'emotion'
+
+const color = 'darkgreen'
+
+render(
+  <div
+    className={css`
+      background-color: hotpink;
+      &:hover {
+        color: ${color};
+      }
+    `}
+  >
+    This has a hotpink background.
+  </div>
+)
+```
+
+#### Object Styles
+
+```jsx
+// @live
+import { css } from 'emotion'
+
+const color = 'darkgreen'
+
+render(
+  <div
+    className={css({
+      backgroundColor: 'hotpink',
+      '&:hover': {
+        color
+      }
+    })}
+  >
+    This has a hotpink background.
+  </div>
+)
+```
+
+#### Array of Object Styles
+
+```jsx
+// @live
+import { css } from 'emotion'
+
+const color = 'darkgreen'
+const isDanger = true
+
+render(
+  <div
+    className={css([
+      {
+        backgroundColor: 'hotpink',
+        '&:hover': {
+          color
+        }
+      },
+      isDanger && {
+        color: 'red'
+      }
+    ])}
+  >
+    This has a hotpink background.
+  </div>
+)
+```
+
+### Global Styles
+
+`injectGlobal` injects styles into the global scope and is useful for applications such as css resets or font faces.
+
+```jsx
+import { injectGlobal } from 'emotion'
+
+injectGlobal`
+  * {
+    box-sizing: border-box;
+  }
+  @font-face {
+    font-family: 'Patrick Hand SC';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Patrick Hand SC'),
+      local('PatrickHandSC-Regular'),
+      url(https://fonts.gstatic.com/s/patrickhandsc/v4/OYFWCgfCR-7uHIovjUZXsZ71Uis0Qeb9Gqo8IZV7ckE.woff2)
+        format('woff2');
+    unicode-range: U+0100-024f, U+1-1eff,
+      U+20a0-20ab, U+20ad-20cf, U+2c60-2c7f,
+      U+A720-A7FF;
+  }
+`
+```
+
+### Animation Keyframes
 
-### Demo Sandbox
+`keyframes` is used to define a single [`@keyframes` CSS at-rule](https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes). The generated rule name is returned and can be used in your styles anywhere an [animation-name](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-name) is accepted.
```

Comment:
This feels a bit too technically correct, I don't care about where an animation-name is accepted in the css spec, I care that i have an animation name. `@keyframes` CSS at-rule also seems like strange language like yes, it's an at-rule but as a consumer of keyframes, why do I care? 

Also, it says `generated rule name`, is it actually a rule name? I'd call it an animation name. Is there a name for it in the spec? Is the spec really what consumers care about anyway?